ydb/core/persqueue/ut/resources/counters_topics.html

```diff
@@ -31,6 +31,20 @@
                         name=topic.reserve.used_bytes: 0
                         name=topic.storage_bytes: 0
 
+                        consumer=$without_consumer:
```

Comment:
This is not what we want to see. No label consumer is possible solution. But some sensors are useless in reading without consumer. Sensor topic.partition.committed_end_to_end_lag_milliseconds_max is undefined for example.